### ISSUE TITLE
Is it possible to translate actor provided steps?

### ISSUE THREAD

**Issue by abner** [Codeception/CodeceptJS#184]
Threre is a channel where i would ask few questions about codecept to you?

I work as a System Analyst in the major Governament IT Company in  Brasil, called [SERPRO](http://www.serpro.gov.br/). Actually i'm working  as a consultant in web frontend technologies (at this time AngularJS, Angular 2 [both Javascript and Typescript])   and yesterday casually i found the codecept project and quickly tried it and liked so much.

I would like to use the navigation helper methods in Brazillian Portuguese, so this tests scenarios will serve as our stories documentation too. 

I just came out with this setup:

1 - In the codecept.json file i include a step file like this:

```
"include": {
    "Eu": "./e2e/steps/steps_file.js"
  },
```

> `Eu` is `I` in portuguese

And in the steps_file i created proxy methods in portuguese just calling the corresponding methods:

```
 estouNaPagina: function(url) {
      return this.amOnPage(url);
 },
 
vejo: function(text) {
   return this.see(text);
},

naoEstouEmAplicacaoAngular: function() {
    return this.amOutsideAngularApp();
 }
```

There is some path into the codeceptjs to achieve the translation of all the helpers methods codecept already provides?

I was wondering if an approach like define a mapping json and run some generator to build the steps would be a good way of doing this. 

What do you think?

**Comment by DavertMik** [comment]
Sure! Actually I love the idea. Probably it won't be as meaningful as Cucumber scenarios (it is still JavaScript, anyway), but this mapping looks like a really good idea. To be honset, I had a plan to integrate Cucumber scenarios for high-level business expectations, they could also be used for translations. 

I think this can be implemented patching `lib/actor` and `lib/step` files. 

Step should be able to make different properties for helper method and step name itself:

```js
constructor(helper, name) {
    this.name = name;
    this.helperMethod = name;
}
// inside run() we change to helperMethod
result = this.helper[this.helperMethod].apply(this.helper, this.args);
```

then we can easily change step name:

```
step.name = 'myNewName'; // this won't affect test exectution
```

as for `lib/actor` you can adjust it to change step names by mappings you provide somewhere:

```js
module.exports = function (obj) {
  obj = obj || {};

  let helpers = container.helpers();
  Object.keys(helpers)
    .map(function (key) { return helpers[key];})
    .forEach((helper) => {
      methodsOfObject(helper, 'Helper')
      .filter((method) => {
        return method !== 'constructor' && method[0] !== '_';})
      .forEach((action) => {
        obj[action] = function () {
          let args = arguments;
          recorder.addStep(let step = new Step(helper, action), args);
          step.name = corresponding.mapping.name; // taken from some config....
          return recorder.promise();
        };
      });
    });

  return obj;
};

```

**Comment by abner** [comment]
just submitted a pull request for this #189

**Comment by DavertMik** [comment]
Implemented in #189 
Thanks!
